client/src/component/Title/Title.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import 'font-awesome/css/font-awesome.min.css'
+import './Title.css'
+
+const Title = () => {
+    return (
+        <article className="Title" id="Header">
+            <section className="container">
+                <header>HACKMERCED V</header>
+                <div>20/20/2020</div>
+
+                <div class="box">
+                    <a class="button" href="#popup1">Join the E-mail list</a>
+                </div>
+
+                <div id="popup1" class="overlay">
+                    <div class="popup">
+            <h2>Join our E-Mail list</h2>
+            <form>
+                 <input type = "text" id = "name" placeholder = "Your Name"></input>
+                 <input type = "text" id = "email" placeholder = "Email"></input>
+                 <button type = "submit" id = "sendEmail">submit</button>
+            </form>
+                        <a class="close" href="#">&times;</a>
```

Comment:
In your CSS and in your JSX, try to be more descriptive and direct to what the `id`, `class`, or `className` are in relation to eg. `App-title`, `Title-box`, `mailingList-popUp`, `mailingList-email` etc..
It makes it a lot easier when we are in production to target down bugs and locate any conflicting styling or such.